tasks/internal/request-and-upload-signature/tests/test-request-and-upload-signature-batch-signer.yml

```diff
@@ -0,0 +1,168 @@
+                            "manifest_digest": "digest3",
+                            "signed_claims": {
+                              "redhate2etesting": "signed3"
+                            }
+                          },
+                          {
+                            "claim_file": "cf4",
+                            "manifest_digest": "digest4",
+                            "signed_claims": {
+                              "redhate2etesting": "signed4"
+                            }
+                          }
+                        ],
+                        "created": "2025-10-23T10:54:09.191275Z",
+                        "errors": [],
+                        "pub_task_id": "13",
+                        "repo": "repo1",
+                        "request_id": "f1a99acf-1372-483c-977a-fe89fdd14731",
+                        "request_received_time": "2025-10-23T10:54:10.459475",
+                        "requested_by": "hacbs-testing",
+                        "signing_server_average_time_secs": 0.19742716666666668,
+                        "signing_status": "success"
+                      },
+                      "msg_id": "2025-50bb9dcf-9d54-41f6-b19f-e418ed8bc615",
+                      "timestamp": 1761216851,
+                      "topic": "/topic/VirtualTopic.eng.robosignatory.konflux.sign.13",
+                      "username": "1001030000"
+                    },
+                    {
+                      "amq6100_originalDestination": "topic://VirtualTopic.eng.robosignatory.konflux.sign.13",
+                      "content-type": "text/plain",
+                      "amq6100_destination": "queue://Consumer.hacbs-testing.13.VirtualTopic.eng.robosignatory.konflux.sign.13"
+                    }
+                  ]
+                ],
+                "operation": {
+                  "digests": [
+                    "digest1",
+                    "digest2",
+                    "digest3",
+                    "digest4"
+                  ],
+                  "references": [
+                    "ref1",
+                    "ref2",
+                    "ref3",
+                    "ref4"
+                  ],
+                  "signing_key_names": [
+                    "sigkey1",
+                    "sigkey2"
+                  ],
+                  "signing_keys": [
+                    "37036A73",
+                    "37036A74"
+                  ],
+                  "task_id": "13",
+                  "requester": null
+                },
+                "signing_keys": [
+                  "37036783"
+                ]
+              }
+              EOF
+              touch "$(workspaces.data.path)/mock_pubtools-pyxis-upload-signatures.txt"
+              touch "$(workspaces.data.path)/signing_response.json"
+              cat "$(workspaces.data.path)/mocked_signing_response"
+    - name: run-task
+      taskRef:
+        name: request-and-upload-signature
+      params:
```

Comment:
Shouldn't this be passing signer_type as batch?

Reply:
fixed.